lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/Platform.java

```diff
@@ -7,25 +7,56 @@ public final class Platform {
   private static final String MAC = "mac";
   private static final String WINDOWS = "windows";
 
+  private static final OS OPERATING_SYSTEM = detectOperatingSystem();
+
   private Platform() {}
 
-  public static String getOsName() {
-    return System.getProperty(OS_NAME);
+  public enum OS {
```

Comment:
Shouldn't the `Platform` class itself be the `enum` instead of declaring new `OS` enum?

Reply:
Merged `Platform` and `Platform.OS` into one class in e474e39247e955ef1dfae57a8bdc47c11f4dcea7 - feel free to revert, if it doesn't look satisfactory.